spec/controllers/lists_controller_spec.rb

```diff
@@ -0,0 +1,198 @@
+require 'rails_helper'
+
+begin
+  ListsController
+rescue
+  ListsController = nil
+end
+
+RSpec.describe ListsController, :type => :controller do
```

Comment:
The reason I separated these tests from the given feature tests was because I wanted to directly test the controller actions independent of the view pages and did not need capybara for this. Since the feature tests make use of capybara and is testing both the view page and controller actions, it made sense to me to separate the two.